changelog/upgrade-notes.md

```diff
@@ -33,6 +33,10 @@ Users compiling their code with the library should note that the following typed
 * ol.style.RegularShapeRenderOptions to ol.RegularShapeRenderOptions
 * ol.style.StyleFunction to ol.StyleFunction
 
+#### KML format now uses URL()
+
+The URL constructor is supported by all modern browsers, but not by older ones, such as IE. To use the KML format in such older browsers, a URL polyfill will have to be loaded before use.
```

Comment:
I wonder if this will lead to lots of questions / inconvenience? I still think we should consider using https://www.npmjs.com/package/url-parse It should simply work on the latest IE out of the box IMHO?